www/js/components/ToggleSwitch.tsx

```diff
@@ -11,6 +11,7 @@ const ToggleSwitch = ({ value, buttons, ...rest }: SegmentedButtonsProps) => {
       buttons={buttons.map((o) => ({
         value: o.value,
         icon: o.icon,
+        accessibilityLabel: o.value,
```

Comment:
Side-note that `o.value` here is not really meant to be user-facing text; it's moreso a string used as key for the different toggle options. (e.g. `"details"` /`"graph"`, `"user"` /`"aggregate"`, `"labeled"` / `"detected"`)

I know we don't have translations for those things right now so this is better than nothing, but ideally we'd have dedicated labels